src/Calendar/Navigation.jsx

```diff
@@ -16,6 +16,14 @@ import { formatMonthYear as defaultFormatMonthYear } from '../shared/dateFormatt
 import { isView, isViews } from '../shared/propTypes';
 
 export default class Navigation extends PureComponent {
+  componentDidMount() {
```

Comment:
Having a global keyboard listener for the calendar might be a good idea if you need to handle all keyboard shortcuts in a single place. Since the parent of the calendar is a div (a non-interactive element adding **onKeyDown** on the parent is accessibility anti-pattern (unless you want to make it into a button or give it role of "button")).